commit f1eeef945d49c72eb13654bd30b5904e89b4626f
Author: Florian Weimer <fweimer@redhat.com>
Date:   Fri Dec 10 21:34:30 2021 +0100

    elf: Use errcode instead of (unset) errno in rtld_chain_load

diff --git a/elf/rtld.c b/elf/rtld.c
index 6b0d6107801b2f44..6bbb373c5743cb99 100644
--- a/elf/rtld.c
+++ b/elf/rtld.c
@@ -1143,7 +1143,7 @@ rtld_chain_load (struct link_map *main_map, char *argv0)
 		     rtld_soname, pathname, errname);
   else
     _dl_fatal_printf("%s: cannot execute %s: %d\n",
-		     rtld_soname, pathname, errno);
+		     rtld_soname, pathname, errcode);
 }
 
 static void
